learning_examples_py/migrations_cpswap/listen_logsubscribe.py

```
@@ -0,0 +1,131 @@
+
+"""
+
+
+import json
+import asyncio
+import os
+
+from dotenv import load_dotenv
+from solana.rpc.async_api import AsyncClient
+from solders.pubkey import Pubkey
+from solana.rpc.async_api import AsyncClient
```

Ruff (0.12.2)

Redefinition of unused `AsyncClient` from line 16

Remove definition: `AsyncClient`

(F811)